setup.cfg

```diff
@@ -13,7 +13,7 @@ classifiers =
 [options]
 include_package_data = true
 install_requires =
-    dask[dataframe]!=2021.5.1,!=2021.6.0,<2023.9.2  # gh475 - 2021.5.1 and 2021.6.0 broke ci, omit those versions
+    dask!=2021.5.1,!=2021.6.0,<2023.9.2  # gh475 - 2021.5.1 and 2021.6.0 broke ci, omit those versions
```

Comment:
No, this was correct with the square brackets. This is for PyPI packages. The environment yaml was referring to conda packages.

Reply:
Ah ok, thanks. Fixed.